pandas/_libs/tslibs/frequencies.pyx

```diff
@@ -156,16 +156,16 @@ cpdef get_freq_code(freqstr):
         if is_integer_object(freqstr[0]) and is_integer_object(freqstr[1]):
             # e.g., freqstr = (2000, 1)
             return freqstr
+        elif is_integer_object(freqstr[0]):
```

Comment:
can you update the doc-string to add Raises (TypeError)

Reply:
Updated and green